pipeline.py

```diff
@@ -0,0 +1,66 @@
+import pmu,common
+from tabulate import tabulate
+import itertools,sys,re
+
+def pipeline_view(log,depths):
+  out_pipe=open(log.replace('csv','pipeline.log'),'w');
+  pipe_list=common.file2lines(log)
+  event_list=[]
+  depth_list=[]
+  lines=[]
+  for i in depths:
+    event_list.append(depths[i][0])
+    depth_list.append(int(depths[i][1]))
+  issued_name=depths['issued'][0] 
+  issued_index=[i for i,item in enumerate(event_list) if issued_name in item][0]
+  lines=get_search_list(pipe_list,event_list[0])
```

Comment:
add inline comments to make this readable 
Thinks on min comments for other developers who may need to tweak your code later.